### ISSUE TITLE
The code has dim mismatching problem

### ISSUE THREAD

**Issue by lauranceair** [orobix/retina-unet#37]
change the parameters in keras.json can't solve the problem , maybe is the vision problem? who can tell me!  thx a lot!
